beacon-chain/db/deposits.go

```diff
@@ -22,19 +22,20 @@ var (
 
 // InsertDeposit into the database. If deposit or block number are nil
 // then this method does nothing.
-func (db *BeaconDB) InsertDeposit(ctx context.Context, d *pb.Deposit, blockNum *big.Int) {
+func (db *BeaconDB) InsertDeposit(ctx context.Context, d *pb.Deposit, blockNum *big.Int, depositRoot [32]byte) {
 	ctx, span := trace.StartSpan(ctx, "BeaconDB.InsertDeposit")
 	defer span.End()
 	if d == nil || blockNum == nil {
 		log.WithFields(logrus.Fields{
-			"block":   blockNum,
-			"deposit": d,
+			"block":        blockNum,
+			"deposit":      d,
+			"deposit root": depositRoot,
```

Comment:
Can this be formatted nicely? 

```suggestion
			"deposit root": hex.EncodeToString(depositRoot),
```

By the way, I have found that encoding/hex is much faster than fmt.Sprintf

Reply:
thanks, will use it in the future